core/src/util/builders.cpp

```diff
@@ -4,9 +4,32 @@
 #include "rectangle.h"
 #include "geom.h"
 
+#include <memory>
+
 static auto& NO_TEXCOORDS = *(new std::vector<glm::vec2>); // denotes that texture coordinates should not be used
 static auto& NO_SCALING_VECS = *(new std::vector<glm::vec2>); // denotes that scaling vectors should not be used
 
+void* alloc(void* _userData, unsigned int _size) {
+    return malloc(_size);
+}
+
+void* realloc(void* _userData, void* _ptr, unsigned int _size) {
+    return realloc(_ptr, _size);
```

Comment:
I removed all realloc from fontstash-es because they were really slow, I think it could be nice allocate a memory chunk at the beginning and test in this method if the size goes over it, then just use malloc in that case.